extensions/resource-deployment/src/test/ui/validation/validations.test.ts

```diff
@@ -0,0 +1,191 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import 'mocha';
+import * as should from 'should';
+import { GreaterThanOrEqualsValidation, IntegerValidation, LessThanOrEqualsValidation, RegexValidation, Validation, ValidationType } from '../../../ui/validation/validations';
+
+suite('Validation', () => {
```

Comment:
Why change this to tdd? We use bdd for all our other extension tests - it would be preferable to keep things consistent.